pontoon/base/models.py

```diff
@@ -25,12 +25,72 @@
 log = logging.getLogger('pontoon')
 
 
+class UserTranslationsManager(models.Manager):
+    """
+    Provides various method to interact with larger sets of translations and their stats for user.
+    """
+
+    def _changed_translations_count(self, args):
+        """
+        Helper method, returns expression object which allows us to annotate querysets
+        with counts of translations.
+        """
+        translation_query = (
+            ~Q(translation__string=F('translation__entity__string'))
+            & ~Q(translation__string=F('translation__entity__string_plural'))
+            & Q(translation__user__isnull=False)
```

Comment:
Why did you add this in? It shouldn't be necessary because we're filtering translations that are related to the users we're querying for, so it's implicit that we only operate on translations that have a user assigned.

Did you see a problem with your local testing which led you to add this in?